src/GoProControl.cpp

```diff
@@ -495,6 +495,30 @@ bool GoProControl::isConnected(const bool silent)
   }
 }
 
+  //ADD DAMIEN
+void GoProControl::setConnection()
```

Comment:
shouldn't it be getConnection()? instead of set?

Reply:
yes why not, it get and it set connection :)
I corrected it